eth-contracts/contracts/ClaimsManager.sol

```diff
@@ -199,27 +199,27 @@ contract ClaimsManager is InitializableV2 {
     /**
      * @notice Set the ServiceProviderFactory address
      * @dev Only callable by Governance address
-     * @param _spFactory - address for new ServiceProviderFactory contract
+     * @param _serviceProviderAddress - address for new ServiceProviderFactory contract
```

Comment:
could this be changed to _serviceProviderFactoryAddress? so we don't conflate a service provider user vs the factory contract

Reply:
+1

Reply:
fixed